lambda-multi-svc/README.md

```diff
@@ -65,9 +65,9 @@ aws iam attach-role-policy \
 Then, allow Proton to use that role to provision resources for your services' continuous delivery pipelines:
 
 ```bash
-aws proton-preview update-account-roles \
+aws proton-preview update-account-settings \
   --region us-west-2 \
-  --account-role-details "pipelineServiceRoleArn=arn:aws:iam::${account_id}:role/ProtonServiceRole"
+  --pipeline-service-role-arn "pipelineServiceRoleArn=arn:aws:iam::${account_id}:role/ProtonServiceRole"
```

Comment:
should it be `--pipeline-service-role-arn "arn:aws:iam::${account_id}:role/ProtonServiceRole"`?